<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>259</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>705</width>
    <height>259</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyQRCode</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/appLogo/icon.png">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 1, 255);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleApp">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 1, 255);
color: rgb(239, 219, 255);</string>
    </property>
    <property name="text">
     <string>PyQRCode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 0, 94);
color: rgb(251, 226, 255);</string>
    </property>
    <property name="text">
     <string>Generate QR Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readfromImageBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 0, 94);
color: rgb(251, 226, 255);</string>
    </property>
    <property name="text">
     <string>Read From Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readCameraBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 0, 94);
color: rgb(251, 226, 255);</string>
    </property>
    <property name="text">
     <string>Read Using Camera</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
selection-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
